<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongCreation</class>
 <widget class="QGroupBox" name="SongCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>135</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>135</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>135</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Выбор песни</string>
  </property>
  <widget class="QLineEdit" name="name_input">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>10</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>40</number>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Название:</string>
   </property>
  </widget>
  <widget class="QLabel" name="file_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Путь к файлу:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file_input">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="choose_file_btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>131</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
